tasks/eva_2130/remove_duplicates_in_study.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+import pymongo
+from urllib.parse import quote_plus
+
+
+def find_duplicates_and_remove_them(mongo_user, mongo_password, mongo_host, mongo_database, assembly_accession,
+                                    contig_list, study_list, dry_run):
+    duplicates_to_remove_commands = []
+    with get_mongo_connection_handle_url(
+            username=mongo_user,
+            password=mongo_password,
+            host=mongo_host
+    ) as accessioning_mongo_handle:
+        sve_collection = accessioning_mongo_handle[mongo_database]["submittedVariantEntity"]
+        cursor = sve_collection.find({
+            "seq": assembly_accession, "contig": {"$in": contig_list}, "study": {"$in": study_list}
+        })
```

Comment:
maybe do the same trick to avoid cursor timeouts? if this is a long running query it will benefit from it.

Reply:
For a 1000 variants I don't think it will timeout but better safe than sorry.